packages/manager/src/featureFlags.ts

```diff
@@ -48,11 +48,16 @@ interface GeckoFlag {
   ga: boolean;
 }
 
+interface AclpFlag {
+  enabled: boolean;
+}
+
```

Comment:
```suggestion
interface AclpFlag {
  beta: boolean;
  enabled: boolean;
}

```

Since we've recreated the flag to return a JSON object, we can include a beta boolean to be able to toggle UI changes when the feature is in beta. See the PrimaryNav comment for an example.